students/BRACH_Jakub/lesson_03_Functions/Calculator.py

```diff
@@ -0,0 +1,93 @@
+def SplashScreen():
```

Comment:
method naming is bad, it should be lowercase, from PEP8:
> Function names should be lowercase, with words separated by underscores as necessary to improve readability.

Also do not forget to apply that to naming files.

At least it's consistent in this file.